keps/sig-auth/1687-hierarchical-namespaces-subproject/README.md

```diff
@@ -0,0 +1,152 @@
+any review.
+-->
+
+<!--
+A table of contents is helpful for quickly jumping to sections of a KEP and for
+highlighting any additional information provided beyond the standard KEP
+template.
+
+Ensure the TOC is wrapped with
+  <code>&lt;!-- toc --&rt;&lt;!-- /toc --&rt;</code>
+tags, and then generate with `hack/update-toc.sh`.
+-->
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [Goals](#goals)
+- [Proposal](#proposal)
+- [Risks and Mitigations](#risks-and-mitigations)
+- [Graduation Criteria](#graduation-criteria)
+- [Alternatives](#alternatives)
+  - [Absorb Hierarchical Namespace Controller Into Another Project](#absorb-hierarchical-namespace-controller-into-another-project)
+  - [Hierarchical Namespace Controller Continues As A Non-Sig Project](#hierarchical-namespace-controller-continues-as-a-non-sig-project)
+  - [Abandon Hierarchical Namespace Controller](#abandon-hierarchical-namespace-controller)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+
+## Summary
+
+[Hierarchical Namespace Controller](https://github.com/kubernetes-sigs/multi-tenancy/tree/master/incubator/hnc) makes it easier for you to create and manage namespaces in your cluster. For example, you can create a hierarchical namespace under your team's namespace, even if you don't have cluster-level permission to create namespaces, and easily apply policies like RBAC and Network Policies across all namespaces in your team (e.g. a set of related microservices).
+
+You can read more about hierarchical namespaces in the [HNC Concepts doc](https://docs.google.com/document/d/1R4rwTweYBWYDTC9UC-qThaMkPk8gtflr_ycHOOqzBQE/edit).
+
+
+## Motivation
+
+The Hierarchical Namespace Controller is currently being developed within the Multi-Tenancy Working Group, (of which Sig-Auth the sponsor). As Working Groups are not meant to own code, and Hierarchical Namespace Controller is nearly to MVP status, a permant home is required. Additionally, having a permanent home for Hierarchical Namespace Controller prior to officially releasing will prevent cumbersome migrations of client libraries if a move were to happen at a later time.
+
+
+## Goals
+
+Establish a new repository and permanent home for Hierarchical Namespace Controller at github.com/kubernetes-sigs/hierarchical-namespace-controller to be maintained by the open source Kubernetes community and governed as a subproject of sig-auth.
+
+
+## Proposal
+
+The current multi-tenancy repository will be transferred directly. Subsequent pull requests will then remove HNC from the multi-tenancy repository and reorganize the new Hierarchical Namespace Controller repository as needed. The new source control location will become the authoritative source of truth for all issues and pull requests. As an independent subproject under sig-auth, Hierarchical Namespace Controller will continue to maintain the Apache license hosted [here](https://github.com/kubernetes-sigs).
```

Comment:
In sig-auth, @tallclair mentioned that we should use -incubator until a project becomes a community recommended solution, at which point we would graduate it to -sigs. Thoughts on going through incubator first?

Reply:
Apparently that process is now deprecated: https://github.com/kubernetes/community/blob/master/incubator.md

> IMPORTANT - The Kubernetes Incubator process is now deprecated and has been superseded by Kubernetes subprojects

Reply:
`-incubator` is deprecated: https://github.com/kubernetes/community/blob/master/incubator.md#important---the-kubernetes-incubator-process-is-now-deprecated-and-has-been-superseded-by-kubernetes-subprojects

Reply:
Well that settles that.